src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/InPrisonInterestsResourceMapper.kt

```diff
@@ -23,4 +25,6 @@ interface InPrisonInterestsResourceMapper {
   @Mapping(target = "updatedAt", source = "lastUpdatedAt")
   @Mapping(target = "updatedAtPrison", source = "lastUpdatedAtPrison")
   fun toInPrisonInterestsResponse(inPrisonInterests: InPrisonInterests?): InPrisonInterestsResponse?
+
+  fun toUpdateInPrisonInterestsDto(request: UpdateInPrisonInterestsRequest, prisonId: String): UpdateInPrisonInterestsDto
```

Comment:
The mapper methods (and tests) are really simple at this level, since we're simply mapping to DTOs that have the same fields as the update request objects.

The more complicated update logic is within the entity mappers, which hasn't been changed in this PR.